databuilder/models/table_metadata.py

```diff
@@ -117,7 +117,7 @@ def _create_record_iterator(self) -> Iterator[RDSModel]:
 
 
 # TODO: this should inherit from ProgrammaticDescription in amundsen-common
-class DescriptionMetadata:
+class DescriptionMetadata(GraphSerializable):
```

Comment:
hey @youngyjd , @crazy-2020 recently added https://github.com/amundsen-io/amundsendatabuilder/blob/master/databuilder/models/table_serializable.py#L10 to allow databuilder for mysql, could you make it as subclass as well.

Reply:
actually will let @crazy-2020 comment

Reply:
thanks @feng-tao. I think it's fine to inherit only graph serializable in DescriptionMetadata. rds models do not have a single generic description model used for multiple entities.